static/src/javascripts.flow.archive/lib/getPrivacyFramework.js

```diff
@@ -11,7 +11,7 @@
  */
 
 // @flow strict
-import { isInUsa } from 'projects/common/modules/commercial/geo-utils.js';
+import { isInUsa } from 'projects/common/modules/commercial/geo-utils';
```

Comment:
It's usually fine to leave the Flow archived files as is, no need to update them 🙂

Reply:
There was an error on the Team City build because of the broken file link so we fixed them up, but that's good to know for the future, thank you! 🙂